extensions/blocks/contact-form/variations.js

```diff
@@ -208,4 +214,6 @@ const variations = [
 	},
 ];
 
+const variations = availableVariations.filter( Boolean );
```

Comment:
I'm not sure how in Jetpack, but we usually use the `compact` function from lodash in and around Gutenberg. Fine to keep as is, nothing wrong with it

Reply:
Oh, that's a good point. It does seem a bit cleaner. I just committed d6e89fe42810d661e3bbdd0b77e070fc99f7e699 for this.